app/views/content/is-teaching-right-for-me/all-subjects.md

```diff
@@ -0,0 +1,299 @@
+
+## Mathematics 
+
+<a href="https://getintoteaching.education.gov.uk/funding-and-support/scholarships-and-bursaries?funding_widget%5Bsubject%5D=maths&button=#funding-widget">Tax-free scholarships of £30,000 and bursaries of £28,000 are available for eligible trainee maths teachers</a>.
+
+<a href="/is-teaching-right-for-me/maths">You can learn more about teaching maths</a>. 
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=G1&visa_status=false">Search for maths teacher training courses</a>.
+
+## Modern Languages
+
+<a href=">https://getintoteaching.education.gov.uk/funding-and-support/scholarships-and-bursaries?funding_widget%5Bsubject%5D=languages&button=#funding-widget">Tax-free bursaries of £25,000 are available for all eligible trainee modern languages teachers</a>.
+
+As a modern languages teacher you'll help pupils learn a new language and open them up to a different culture.
+
+
+<a href="/is-teaching-right-for-me/languages">You can learn more about teaching modern languages</a>. 
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=24&visa_status=false">Search for modern languages teacher training courses</a>.
+
+## Music 
+
+<a href="https://getintoteaching.education.gov.uk/funding-and-support/scholarships-and-bursaries?funding_widget%5Bsubject%5D=music&button=#funding-widget">Tax-free bursaries of £10,000 are available for eligible trainee music teachers</a>. 
+
+As a music teacher you'll inspire pupils to develop a love of music and their talent as musicians. 
+
+You can <a href="https://www.thenational.academy/teachers/programmes/music-secondary-ks3/units">explore what teaching a music lesson could look like</a>.
+
+ <a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=W3&visa_status=false">Search for music teacher training courses</a>.
+
+## Philosophy 
+
+As a philosophy teacher you'll teach pupils ways of thinking about ethics, time and meaning. Help them explore the big questions of life and morality. 
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=P1&visa_status=false">Search for philosophy teacher training courses</a>.
+
+## Physical education 
+
+As a P.E teacher you'll help pupils to develop motor skills and an interest for physical activity and fitness. 
```

Comment:
@caitlin-reidy PE without .

Reply:
@caitlin-reidy what sort of sports can they teach? Are they going to train the next Jack Grealish or Mary Earps (or whoever the most popular footballers or other sports people are?)  or encourage someone in a sport to go on the Olympics! - What/how could they get children excited about PE?